kitsune/search/jinja2/search/results.html

```diff
@@ -33,23 +33,6 @@
         {% if product %}
```

Comment:
Do we need any of this `base_doctype_url` logic here anymore? Doesn't look like it's used now...